src/univers/versions.py

```diff
@@ -219,6 +219,64 @@ def is_valid(cls, string):
         except ValueError:
             return False
 
+    @property
+    def major(self):
+        return self.value and self.value.major
```

Comment:
Nice trick !

Reply:
That's more of an idiom to me! Though in earnest I am always split in abusing this shortcutting of "and" and "or" and their returning of a value as used here ... it can become hard to read unless used for very simple checks like this one.